web/src/main/java/se/inera/intyg/webcert/web/web/controller/legacyintegration/FragaSvarUthoppController.java

```diff
@@ -92,14 +104,23 @@ public Response redirectToIntyg(@Context UriInfo uriInfo,
         @PathParam("intygId") String intygId,
         @QueryParam("enhet") String enhetHsaId) {
 
-        super.validateParameter("type", type);
-        super.validateParameter("intygId", intygId);
-        super.validateAuthorities();
-        this.validateAndChangeEnhet(intygId, type, enhetHsaId);
+        if (reactPilotUtil.useReactClientFristaende(webCertUserService.getUser(), type)) {
```

Comment:
Look at moving this login into "buildRedirectResponse". 

The logic to validate parameters and unit etc should always be executed.

Reply:
Fixed